src/main/resources/lang.yml

```diff
@@ -38,6 +38,11 @@ achievement-already-received: "The player PLAYER has already received this achie
 achievement-not-found: "The specified achievement was not found in Commands category."
 achievement-given: "Achievement given!"
 
+# Related to /aach add.
+error-value: "The value VALUE must to be an integer!"
+achievement-increase: "Achievement ACH increase by AMOUNT for PLAYER!"
```

Comment:
Should we talk about "statistics" here as well? Also I would have written "increase**d**".